scripts/ci/automation_full_test.py

```diff
@@ -99,6 +107,97 @@
         }
 
 
+def run_git_command(cmd):
+    out = subprocess.run(cmd)
+    if out.returncode:
+        raise RuntimeError(f"{cmd} failed")
+
+
+def git_restore(file_path):
+    if not file_path:
+        return
+    logger.info(f"git restore {file_path}")
+    out = subprocess.Popen(["git", "restore", file_path], stdout=subprocess.PIPE, stderr=subprocess.STDOUT)
+    stdout, err = out.communicate()
+    if stdout:
+        logger.info(stdout)
+    if err:
+        logger.warning(err)
+
+
+def git_push(message):
+    out = subprocess.Popen(["git", "status"], stdout=subprocess.PIPE, stderr=subprocess.STDOUT)
+    stdout, _ = out.communicate()
+    if "nothing to commit, working tree clean" in str(stdout):
+        return
+    try:
+        run_git_command(["git", "add", "*/command_modules/*"])
+        run_git_command(["git", "commit", "-m", message])
+    except RuntimeError as ex:
+        raise ex
+    retry = 3
+    while retry >= 0:
+        try:
+            run_git_command(["git", "fetch", "azclibot"])
+            run_git_command(["git", "pull", "--rebase"])
+            run_git_command(["git", "push"])
+
+            logger.info("git push all recording files")
+            break
+        except RuntimeError as ex:
+            if retry == 0:
+                raise ex
+            retry -= 1
+            time.sleep(10)
+
+
+def run_azdev(cmd):
```

Comment:
Is it better to rename the function to meaningful name as the function not only support running "azdev"?

Reply:
Replaced with `run_command`